app/controllers/configuration_controller.rb

```diff
@@ -431,54 +359,12 @@ def set_form_vars
         :current => {},
         :key     => 'config_edit__ui4',
       }
-      @edit[:timeprofile_id] = @timeprofile.try(:id)
-      if %w[timeprofile_new timeprofile_copy timeprofile_edit timeprofile_update].include?(params[:action])
-        @edit[:current] = {
-          :description  => @timeprofile.description,
-          :profile_type => @timeprofile.profile_type || "user",
-          :profile_key  => @timeprofile.profile_key,
-          :profile      => {
-            :days  => Array(@timeprofile.days).uniq.sort,
-            :hours => Array(@timeprofile.hours).uniq.sort,
-            :tz    => @timeprofile.tz,
-          },
-          :rollup_daily => @timeprofile.rollup_daily_metrics,
-        }
-        @edit[:all_days]  = @edit.fetch_path(:current, :profile, :days).length == 7
-        @edit[:all_hours] = @edit.fetch_path(:current, :profile, :hours).length == 24
-      end
       show_timeprofiles
     end
     @edit[:new] = copy_hash(@edit[:current])
     session[:edit] = @edit
   end
 
-  def get_form_vars
-    @edit = session[:edit]
-    case @tabform
-    when "ui_1" # Visual Settings tab
-      @edit[:new][:perpage][:grid] = params[:perpage_grid].to_i if params[:perpage_grid]
-      @edit[:new][:perpage][:tile] = params[:perpage_tile].to_i if params[:perpage_tile]
-      @edit[:new][:perpage][:list] = params[:perpage_list].to_i if params[:perpage_list]
-      @edit[:new][:perpage][:reports] = params[:perpage_reports].to_i if params[:perpage_reports]
-      @edit[:new][:display][:theme] = params[:display_theme] unless params[:display_theme].nil?
-      @edit[:new][:display][:bg_color] = params[:bg_color] unless params[:bg_color].nil?
-      @edit[:new][:display][:reporttheme] = params[:display_reporttheme] unless params[:display_reporttheme].nil?
-      @edit[:new][:display][:dashboards] = params[:display_dashboards] unless params[:display_dashboards].nil?
-      @edit[:new][:display][:timezone] = params[:display_timezone] unless params[:display_timezone].nil?
-      @edit[:new][:display][:startpage] = params[:start_page] unless params[:start_page].nil?
-      @edit[:new][:display][:locale] = params[:display_locale] if params[:display_locale]
-      @edit[:new][:display][:compare] = params[:display][:compare] if !params[:display].nil? && !params[:display][:compare].nil?
-      @edit[:new][:display][:drift] = params[:display][:drift] if !params[:display].nil? && !params[:display][:drift].nil?
-    when "ui_3" # Visual Settings tab
```

Comment:
I believe ui_3 is still present, right?  I think the comment here is actually wrong.  Same with the comment for ui_4 below.

Reply:
Yeah ui_3 is still present. The only place I saw this function be called is on this line: 
`get_form_vars if @tabform != "ui_3"`

Not sure if I understood this right or not but will get_form_vars only be called if @tabform is not equal to ui_3? This would mean that the ui_3 block in this function is never entered. Also, the code in these blocks doesn't seem related to the forms on the ui_3 and ui_4 tabs.  The code seems to be referring to a [:display][:compare] value which comes from the ui_1 tab. Not sure if this was just a copy paste error or not.

Reply:
oh ok - I thought get_form_vars would have been called elsewhere, but if not, then this is fine.